reconcile/utils/aws_api.py

```diff
@@ -846,6 +846,8 @@ def _get_assumed_role_client(
         self, account_name: str, assume_role: str, assume_region: str, client_type="ec2"
     ) -> EC2Client:
         session = self.get_session(account_name)
+        if assume_role is None:
```

Comment:
this would also be triggered in the case of a missing `awsInfrastructureManagementAccess` section. I am wondering if we could use an empty string instead of `None` to show that the config is valid, but there are no role to assume.
WDYT @maorfr ?

Reply:
Proposed using an empty string in 1eab41f